### PR TITLE
Create a scrolling background and move the player

### PR DESCRIPTION
**Pull request by louissalin**

### CODE REVIEW

File: chapter-06/start/States/GameplayState.cs [hunk 1]
```diff
@@ -26,6 +37,33 @@ public override void HandleInput()
...
+            if (state.IsKeyDown(Keys.Right))
+            {
+                _playerSprite.MoveRight();
+                KeepPlayerInBounds();
+            }
+        }
+
+        private void KeepPlayerInBounds()
+        {
+            if (_playerSprite.Position.X < 0)
```
Comment (jcapellman) [inline chapter-06/start/States/GameplayState.cs]: Brace use for the conditionals (even single line) - I believe I had established that earlier

File: chapter-06/start/Objects/Base/BaseGameObject.cs [hunk 2]
```diff
@@ -11,11 +11,18 @@ public class BaseGameObject
 
         public int zIndex;
 
+        public int Width { get { return _texture.Width; } }
+        public int Height { get { return _texture.Height; } }
+        public Vector2 Position 
+        { 
+            get { return _position; }
```
Comment (jcapellman) [inline chapter-06/start/Objects/Base/BaseGameObject.cs:18]: Expression Body here is style++ :)

File: chapter-06/start/States/GameplayState.cs (hunk 1 above)
Comment (louissalin) [reply chapter-06/start/States/GameplayState.cs]: I have noticed your braced single line bodies. For some reason it didn't compute in my head that I should follow that style. Fixing.

File: chapter-06/start/Objects/Base/BaseGameObject.cs (hunk 2 above)
Comment (louissalin) [reply chapter-06/start/Objects/Base/BaseGameObject.cs:18]: eh, having the get and the set on the same line was just ugly

File: chapter-06/start/States/GameplayState.cs (hunk 1 above)
Comment (louissalin) [reply chapter-06/start/States/GameplayState.cs]: fixed

**Comment by louissalin** [comment]
you know I almost got to use pattern matching last night. I had added an event type and a pattern match before realizing, unfortunately, that I didn't need it. :(

oh, pattern matching will come! Just not yet...

**Review by jcapellman — APPROVED** [review]

**Pull request merged**